bindings/src/Capgemini.PowerApps.SpecFlowBindings/Steps/LoginSteps.cs

```diff
@@ -38,5 +42,49 @@ private static void CloseTeachingBubbles()
                 closeButton.Click();
             }
         }
+
+        private static void Login(IWebDriver driver, Uri orgUrl, string username, string password)
+        {
+            driver.Navigate().GoToUrl(orgUrl);
+            driver.ClickIfVisible(By.Id("otherTile"));
+
+            bool waitForMainPage = WaitForMainPage(driver);
+
+            if (!waitForMainPage)
+            {
+                IWebElement usernameInput = driver.WaitUntilAvailable(By.XPath(Elements.Xpath[Reference.Login.UserId]), 30.Seconds());
+                usernameInput.SendKeys(username);
+                usernameInput.SendKeys(Keys.Enter);
+                driver.WaitForTransaction();
+
+                IWebElement passwordInput = driver.WaitUntilClickable(By.XPath(Elements.Xpath[Reference.Login.LoginPassword]), 30.Seconds());
+                passwordInput.SendKeys(password);
+                passwordInput.Submit();
+                driver.WaitForTransaction();
```

Comment:
`driver.WaitForTransaction()` checks a flag in JavaScript that's only available within model-driven apps. Won't do anything  on the login pages